ts/helpers/checkobj.ts

```diff
@@ -5,10 +5,18 @@
 
 import * as format from "string-template";
 
-export function CheckValidObject(obj: any, path: string): any {
+export function getValueAtPath(obj: any, path: string): any {
     try {
         return eval(format("obj{0}", path));
     } catch (e) {
         return undefined;
     }
 }
+
+export function checkPath(obj: any, path: string): boolean {
+    if (getValueAtPath(obj, path) === undefined) {
+        return false;
+    } else {
+        return true;
```

Comment:
```suggestion
    return getValueAtPath(obj, path) !== undefined;

```

Reply:
committed the suggestion @shak

Reply:
eek sorry . .didn't you were going to commit it as is .. there was an line break for no reason, remove that please. I'll be more careful next time.